manifests/apache.pp

```diff
@@ -60,6 +60,17 @@
     content => template('katello/pulp-apache-ssl.conf.erb'),
   }
 
+  if $katello_devel::pulp3 {
+    class { 'pulpcore':
```

Comment:
Including `pulpcore` in the Apache config doesn't feel correct. I'd do it in `init.pp`.